src/components/rtoken-setup/governance/GovernanceParameters.tsx

```diff
@@ -32,7 +32,7 @@ const GovernanceParameters = (props: BoxProps) => {
           required: true,
           pattern: numberPattern,
           min: 1,
-          max: 259200, // 3 days
+          max: 50400, // 3 days
```

Comment:
The comment here should be 1 week

Reply:
And in the FormField tag above there lots of mentions of seconds, where it should blocks.